module/VuFindSearch/src/VuFindSearch/Backend/EDS/Base.php

```diff
@@ -269,11 +269,14 @@ protected function parseAutocomplete($msg)
      */
     public function autocomplete($query, $type, $data, $raw = false)
     {
-        // build request
-        $url = $data['url'] . '?idx=' . urlencode($type) .
-            '&token=' . urlencode($data['token']) .
-            '&filters=[{"name"%3A"custid"%2C"values"%3A["' .
-            urlencode($data['custid']) . '"]}]&term=' . urlencode($query);
+        $params = [];
+        $params['idx'] = $type;
+        $params['token'] = $data['token'];
+        $params['filters'] = '[{"name" : "custid", "values": ["'.$data['custid'].'"]}]';
```

Comment:
Might it be better to change:

`$params['filters'] = '[{"name" : "custid", "values": ["'.$data['custid'].'"]}]';`

to:

`$params['filters'] = '[{"name" : "custid", "values": ' . json_encode([$data['custid']]) . '}]';`

...to ensure that the custid value is always appropriately encoded?